101-traffic-manager-virtual-machines/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
This looks like a 201 template for your service as it is a bit more involved than 101 with vms and all. Also consider renaming to 201-traffic-manager-vm-load-balance